cmake/OpenFX.cmake

```diff
@@ -0,0 +1,53 @@
+if(APPLE)
+  set(PLUGINDIR "/Library/OFX/Plugins")
+  set(ARCHDIR "MacOS-x86-64")
```

Comment:
Change this to `MacOS`, the universal binary location. Plug-ins built with this script on an Apple Silicon Mac come out as universal so don't work unless the binaries are in MacOS.